src/main/java/org/javamodularity/moduleplugin/tasks/StartScriptsMutator.java

```diff
@@ -120,4 +131,16 @@ private static void replaceLibsPlaceHolder(Path path, String libText, String pat
             throw new GradleException("Couldn't replace placeholder in " + path);
         }
     }
+
+    private static void removeClasspathArg(Path path, String original, String replacement) {
+        try {
+            String bashScript = Files.readString(path);
```

Comment:
I'd rename `bashScript` to `scriptContent` and `updatedBashScript` to `updatedScriptContent` (as it's used both for Bash and Bat scripts).

PS. Ha, I see it's the same as in the already existing `replaceLibsPlaceHolder` method. Anyway, I think it might be worth improving in both places.